graphrag_sdk/models/litellm.py

```
@@ -73,7 +73,7 @@ def check_valid_key(self, model: str):
                 model=model, messages=messages, max_tokens=10
             )
             return True
-        except:
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)